helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -1229,7 +1230,7 @@ public DeploymentGroupStatus getDeploymentGroupStatus(final String name)
   private List<String> getHosts(final ZooKeeperClient client, final String path) {
     try {
       return Json.read(client.getNode(path).getBytes(), STRING_LIST_TYPE);
-    } catch (JsonMappingException | NoNodeException e) {
+    } catch (JsonMappingException | JsonParseException | NoNodeException e) {
```

Comment:
for future reference it would be good to log something when this unparsable-data-found-but-falling-back-to-empty event happened